README.md

```diff
@@ -76,10 +83,23 @@ mkdir -p ~/geoserver_data
 docker run -d -v $HOME/geoserver_data:/opt/geoserver/data_dir kartoza/geserver
 ```
 
-You need to ensure the ``geoserver_data`` directory has sufficinet permissions
+You need to ensure the ``geoserver_data`` directory has sufficient permissions
 for the docker process to read / write it.
 
+## Setting Tomcat properties
 
+To set Tomcat properties such as maximum heap memory size, create a `setenv.sh` file such as:
+
+```shell
+JAVA_OPTS="$JAVA_OPTS -Xmx1536M -XX:MaxPermSize=756M"
+JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"
```

Comment:
@isuftin The current [README](https://github.com/kartoza/docker-geoserver#storing-data-on-the-host-rather-than-the-container) covers how to make the data dir persistent via a Docker volume, so I just set the GEOSERVER_DATA_DIR environment var in the Dockerfile to the already mentioned path for consistency (`/opt/geoserver/data_dir`). This approach is similar to other images I've seen (e.g. Postgres).